README.md

```diff
@@ -107,6 +107,32 @@ def import_fontawesome_stylesheet():
 Note that due to the way EasyMDE defines the toolbar icons it is not compatible with
 [Wagtail FontAwesome](https://gitlab.com/alexgleason/wagtailfontawesome)
 
+#### Django-compressor example
```

Comment:
suggestion: change to an `h5` (i.e. `#####`) or simply make it bold.
suggestion: given the context, I'd rename it to something like "Using with django-compressor"

Reply:
Thanks! I went over it and updated it.